x-pack/plugins/enterprise_search/public/applications/workplace_search/views/security/security.tsx

```diff
@@ -180,20 +157,25 @@ export const Security: React.FC = () => {
   );
 
   return (
-    <>
-      <SetPageChrome trail={[NAV.SECURITY]} />
-      <FlashMessages />
+    <WorkplaceSearchPageTemplate
+      pageChrome={[NAV.SECURITY]}
+      pageHeader={{
+        pageTitle: PRIVATE_SOURCES,
+        description: PRIVATE_SOURCES_DESCRIPTION,
+        rightSideItems: headerActions,
```

Comment:
**nit (not for this PR)**: What do you think about renaming `rightSideItems` to just `sideItems` or `actions`?
"Right" might not be a correct name in some environments: possibly with rtl languages, on mobile, or screen readers.

Reply:
Sorry still waking up so I just realized I didn't fully answer you. I totally agree with you on the rtl thing - I like `sideItems` personally, `actions` feels a little prescriptive since you can have just plain text or notifications and not buttons there. But yeah I think this is totally worth suggesting to the EUI folks, if you wanna ping them! 👍

Reply:
Ah, didn't know that! Well, ok, then. 🤷